src/commands/submit.ts

```diff
@@ -149,6 +149,8 @@ export default class Submit extends Command {
         data.append('app', fs.createReadStream(fd.mergeWithFolder(zipName)));
         data.append('email', flags.email);
         data.append('categories', JSON.stringify(selectedCategories));
+        data.append('purchaseType', 'buy');
```

Comment:
I don't think this should be handled by the CLI, as we are not sure how the flow for changing this will be later on.
Please remove